Added basic functionality for T-Watch

themelispan opened the pull request:
* Fixes to support older Android Studio (Gradle)
* Added T-Watch feature "Clock"
* Added T-Watch feature "Battery indicator"

themelispan committed:
Added basic functionality for T-Watch

* Fixes to support older Android Studio (Gradle)
* Added T-Watch feature "Clock"
* Added T-Watch feature "Battery indicator"

themelisx commented on Arduino/TWatch 2.0/DisplayTime.ino:
```diff
@@ -0,0 +1,99 @@
+      p = 100;
+    }
+    char percentage[5];
+    memcpy(percentage, " ", 5);
```
@themelispan  memcpy(percentage, " ", 5) should be replaced with memset()

themelisx commented on Arduino/TWatch 2.0/LowEnergy.ino:
```diff
@@ -0,0 +1,43 @@
+      ttgo->displaySleep();
+
+      hasLowEnergy = true;
+      Serial.println("screen off");
```
@themelispan Place all your Serial.println into #IFDEF DEBUG

themelisx commented on Arduino/TWatch 2.0/Notifications.ino:
```diff
@@ -0,0 +1,61 @@
+
+///////////////////////////////////////////////////////////////////// NOTIFICATION FUNCTIONS
+
+void viewNewNotif(){
```
@themelispan Try to make more clear names

themelisx reviewed (changes requested):
Please also check that the BLE works

themelispan committed:
Fixes based on code review comments

@themelisx  Serial prints should be placed into IFDEF debug on a seperate issue/branch
